Sources/MessageInputBarDelegate.swift

```diff
@@ -24,8 +24,14 @@
 
 import Foundation
 
+@objc
 public protocol MessageInputBarDelegate: class {
-
-    func sendButtonPressed(sender: UIButton, textView: UITextView)
-
+    
+    @objc optional func inputBar(_ inputBar: MessageInputBar, didSelectSendButtonWith text: String)
+    
```

Comment:
Nit but do you mind renaming these to `messageInputBar`, I'd like to be more specific.
Do you think `didSelectSendButtonWith` -> `didPressSendButtonWith` is better?
Select makes me think some sort of state was set.

Also, the rest of the MessageKit delegates broke away from the Apple pattern of using the object as the method name. Instead, the method name represents exactly what it does.

IDK which approach is the best to be honest, but bike shedding aside, I think the API should feel similar.